src/snapshot.zig

```diff
@@ -347,24 +351,19 @@ fn processSnapshotFile(gpa: Allocator, snapshot_path: []const u8, maybe_fuzz_cor
     }
 
     // Write out any TOKENS
-    {
+    if (std.mem.indexOf(u8, content.meta, "verbose-tokens") != null) {
         try file.writer().writeAll(Section.TOKENS);
         try file.writer().writeAll("\n");
-        const tokenizedBuffer = parse_ast.tokens;
+        var tokenizedBuffer = parse_ast.tokens;
         const tokens = tokenizedBuffer.tokens.items(.tag);
-        var first = true;
-        for (tokens) |tok| {
-
-            // only write a comma if not the first token
-            if (first) {
-                first = false;
-            } else {
-                try file.writer().writeAll(",");
-            }
-
+        for (tokens, 0..) |tok, i| {
+            const region = tokenizedBuffer.resolve(@intCast(i));
+            const region_str = try std.fmt.allocPrint(gpa, "{}\t", .{region});
+            defer gpa.free(region_str);
+            try file.writer().writeAll(region_str);
             try file.writer().writeAll(@tagName(tok));
+            try file.writer().writeAll("\n");
         }
```

Comment:
Made this optional, it looks for the word `verbose-tokens` in the META section. I figure not all snapshot tests need this level of information, and always printing all the tokens was feeling like unnecessary noice. This way relevant snapshots can include this information.

Reply:
So standard tokens is just printing the name of all tokens and verbose tokens prints the region of all tokens.

Also, if you write less new lines, it might make it more manageable.
I would try only adding new lines after new line tokens. That way the tokens print out mapping to the lines in the original input. Will also make it easier to validate line/column mapping.